cpp/models/ode_seir/model.h

```diff
@@ -63,6 +64,43 @@ class Model : public CompartmentalModel<InfectionState, Populations<InfectionSta
         dydt[(size_t)InfectionState::Recovered] =
             (1.0 / params.get<TimeInfected>()) * y[(size_t)InfectionState::Infected];
     }
+
+    /**
+    *@brief After the simulation is finished and we get a resulting TimeSeries, this function uses the data to compute the reproduction number
+    *at an arbitrary time
+    *@param timept The time at which we want to compute the reproduction number
+    *@param y The TimeSeries. We actually only use the number of Susceptibles at a given time
+    *@returns The reproduction number in the seir model
+    *@see The same functions in the model.h files of the secir and secirvvs models
+    */
+
+    ScalarType get_reproduction_number(Eigen::Index timept, mio::TimeSeries<ScalarType> y)
```

Comment:
```suggestion
    ScalarType get_reproduction_number(Eigen::Index timept, const mio::TimeSeries<ScalarType>& y)
```
Using the whole TimeSeries as argument is not optimal. Otherwise it would be easier to call for the user.
So this would be ok. But otherwise, you should definitly do this with a reference